scripts/jest.setup.ts

```diff
@@ -2,9 +2,16 @@ const chalkFunction = new Proxy((x: string) => x, {
   get: () => chalkFunction
 });
 
-jest.mock('chalk', () => new Proxy({}, {
-  get: () => chalkFunction
-}));
+jest.mock(
```

Comment:
I don't honestly agree with this change, imo it makes this more unreadable

Reply:
Agreed, I'll manually look through the changes and revert some of them.